report/UsabilityTest/UsabilityTest.tex

```diff
@@ -0,0 +1,49 @@
+\section(Usability testing)
+In order to determine if our application is easy to use and fulfils the requirements, we decided to perform usability testing of our application very early in the project.
+However due to the Covid-19 situation, we have been left with no options of performing usability testing in a safe environment and since we have not been able to perform the usability tests in a safe environment, we have been forced to not test the application.
+
+Below is an example of a usability test, that should have been performed to test the usability and flow of the application. 
+The test is intended to test most of the general flow in the application, and test if the app is easy to navigate. 
+
+The test is also designed to not indicate where the different elements are located on the screen.
+This ensures that the elements used in the test should be intuitive to find and interact with. 
+
+\begin{enumerate}
+    \item Open the Application called “Flutter Application Editor”.
+    \item A login screen should be visible. 
+    \item Tap the username field.
+    \item Enter “Graatand” on the keyboard.
+    \item Tap the password field
+    \item Enter “Password” into the password field.
+    \item Tap the login button.
+    \item The application selecting screen should be visible after a few seconds of loading.
+    \item Tap an any of the showing applications
+    \item A new screen 
+    \item Tap the “Show tasks” icon.
+    \item A kanban board should be visible.
+    \item Click the “+” icon.
+    \item The new tasks screens should be shown.
+    \item The create task button should be disabled and non-functional 
+    \item Tap the task name field.
+    \item Enter a name of the task.
+    \item Tap the task description field.
+    \item Enter a description of the task.
+    \item Tap the priority drop down box.
+    \item Select a priority.
+    \item Hide the keyboard.
+    \item Press the “Create Screen” button.
+    \item Tap the screen name field.
+    \item Enter a name for the screen.
+    \item Tap the create screen button.
+    \item A box saying that the screen has been created should be visible within a few seconds.
+    \item Discard the box by pressing the “ok” button.
+    \item The new task screen should be visible again, with all the information entered before.
+    \item There should be some text saying which screen is selected.
+    \item The create task button should now be enabled.
+    \item Tap the create task button.
+    \item A box saying that the task was created successfully should be visible within a few seconds.
+    \item Tap the “ok” button.
+    \item The task screen should be visible
+    \item The newly created task should be visible in the “not started” column.
+    \item Done. 
```

Comment:
Mangler vi ikke nogle steps til screen editoren?

Reply:
Jo, teknisk set gør vi, men jeg valgte ikke at have det med, da det er ikke er relateret til at oprette tasks og jeg syntes derfor det burde være en test for sig selv